### DISCUSSION TITLE
Ineffable Perfection, Devourer of Faces, Confidential M-Class Hivemind Ship

### DISCUSSION THREAD

**Discussion by TallDeekus** [shiptest-ss13/Shiptest#488] [Ideasguy]
_(I am sensing a theme with my ideas as of late, hopefully it is just a phase, right?)_

_(Alas, things are best portrayed by story first...)_
### ALERT, LEVEL 4 CLEARANCE IS REQUIRED BEYOND THIS POINT, CHECKING ACCESS LEVEL NOW.

_**Scanning...**_

_**Scanning...**_

### USER CLEARANCE AUTHORIZED, OPENING FILE.

_Opening Attached Voice Recording..._

I knew in my gut there was something terrible on this rock, but I was too short on funds, the money was too good to be true.
I knew that researcher [DATA EXPUNGED] was hiding something, when he said he was here before, and that he knew where we needed to dig.
"Dislodge the designated object and you will receive your pay on retrieval." he said, "This is delicate work, bring only pickaxes." he said, fuck man...
Confidential work my ass, it fucking ate my boss.

Where do I even fucking begin... Right, myself, this is Danny Reckingcrew, you might have heard of me from [HIGHER CLEARANCE REQUIRED], as part of the Whiteshuttle mercenary gang, I... We were hired for an job that was just described as just 'Excavating lost assets' by a Dr. D [DATA EXPUNGED], we had fallen on hard times and accepted the job without a second thought, that was possibly our last mistake.

When we arrived, we saw a ship lodged into the stone, a crash like that would have rendered it irreparable but the Researcher kept insisting that 'she' was still flyable, and that we only needed to remove the surrounding rock, seemed like an easy job, and personally I was eager to get paid then visit the place with the ladies the crew and I were eying, some respite from the bullshit of the universe, but... Fuck fuck fuck they are gone, their gone!

Focus focus... Breathe... The job was normal until we had removed enough rock to allow the Osprey-class out, when the Doctor invited us inside for something to eat, then proceeded to enter, I was hesitant as always, I have the scars to remind myself to be cautious of hospitality and it saved me, because when my crew entered and I watched I-I-...

They were fucking melting, melting! They kept walking like nothing was bloody happening as their flesh started t-

_The sound of retching can be heard, immediately followed by a coughing fit._

_Afterwards there is just a moment silence, save for an unrhythmic drumming sound that sounds of something drumming on ship hull._

And the Doctor, he just kept walking inside as the walls lost their consistency, taking on a almost foggy glass look, and dripping like slime, I-I-I ran, ran as my crew called for me to join them, to be with them...

I've never flown a shuttle in my life, but this is my only shot, I can hear them outside and it will only be so long before they unlock the airlock, if someone finds this and I am gone, tell Dr. [DATA EXPUNGED] he can [REDACTED] [REDACTED] [REDACTED].

_Addendum: The shuttle and all objects aboard including the voice recording were found intact, but floating abandoned of life in sector [HIGHER CLEARANCE REQUIRED], however a mucus was found in the airlock of the ship that can be described as Light Sea Green, the mentioned ship now classified as Ineffable Perfection was spotted in the [HIGHER CLEARANCE REQUIRED] heading towards the planet [HIGHER CLEARANCE REQUIRED].

### Description

The Ineffable Perfection is what appears to be an Osprey-class ship on the outside, however in actuality is a massive organism of similar biology to that of a slime, wearing the hull as a disguise.

In all terms and purposes, the oddity works just like a normal ship except when a normal unprotected individual walks into it, when this happens the individual is slowly digested and seemingly 'replaced' with a double made out of the same substance as the oddity, this individual behaves and acts almost like their previous counterpart, and recalls all of the memories of their previous life.

However, every individual assimilated into the Oddity loses their individualism and identity, their form looking like a humanoid without any features like Hair, it is also noted that individuals affected by this have reported a short of 'hivemind' with anyone else assimilated, and the ship itself.

It is to note when a slimeperson enters the ship unprotected, they keep most of their identity and appearance, however their color will be forcibly altered on a genetic level to be identical to that of the ship, and they will be assimilated into the hivemind aswell.

Individuals removed from the Oddity for extended periods of time lose motor functions and mental acuity over time, until eventually completely liquifying, this process is considered fatal.

It is also to note that those assimilated into the ship do not have slime cores in their persons, instead they appear to be housed with the ship's 'core'.

Other then the mentioned effects above, assimilated crew physically seem healthier and happier inside 'their' new ship, and will try to encourage new people to join them, it is theorized that this isn't simply hospitality and instead a form of reproduction, as like any slime the ship itself could possibly split and create more of itself when it has enough mass.

The oddity is to be relocated and re-contained after having evaded operatives after questioning.

### Author's Additional Notes

- When the ship is considered 'destroyed', all assimilated crewmembers will perish if it doesn't get repaired in time, they can be revived as normal with slimepeople, if their slime core is relocated that is.
- An individual can be unhiveminded by the removal of their slime core from the ship, this will kill them, but they can be revived without putting it back by inserting the slime core into their person, and preforming revival surgery.
- Slight health regeneration and a positive moodlet is given to assimilated crew just by being in their ship.
- Multiples of the ship will automatically be hiveminded together. (unless they are a different color? Teams mode confirmed?)
- The Digestion should work just like the zombie virus except it cures when leaving the ship before it completes, the damage it causes is both Burn and Genetic.
- The ship has everything a normal ship would have, excluding a Cryobed that people can spawn at.
- While the hivemind is cureable, physical alterations are not, and require _special_ treatment. (Admin intervention or Magic Mirror)
- Do not eat the slime.
- The ship spawns with a Human Suit, to blend in.
- Medbay Chemicals are 'flipped', to account for Slime Biology.

_(I might have missed some ideas but feel free to put your own in the comment section!)_

**Comment by Ms-Mee** [comment]
Yeah this heavily reminds me of Melting Love and seems quite interesting, nothing I have to add currently

**Comment by zestybastard** [comment]
I would make the burn/genetic damage effect on the ship toggleable, in case the ineffable crew decides they want to try a more diplomatic "come and stay a while" route. also, what's the intended playstyle for this and would it count as an antag ship?

**Reply by TallDeekus** [reply]
There are several ideas, such as it being a possible antag spawn with the antag being an 'ai' that controls the ship, that is in actuality the ship itself.

The other idea was it to have it as a ruin spawn.

But in no way I intend for it to be roundstart, unless the lead devs have a plan for Exotic ships like this.

**Reply by zestybastard** [reply]
we might bus it in bc this looks like a Really fun antag that seems to be better balanced than piracy

**Reply by TallDeekus** [reply]
Hold on let me copy paste the objectives I just made in discord:

> X/X total alive converts.

> Get X amount of research points assimilated.

> Assimilate X amount of (Material).

> Place Colonization Spore Nodes on X amount of unique planets.

**Comment by Zevotech** [comment]
How would this affect different races like IPC's and Kepori? Maybe even Vox when they get added?
